src/helpers/signTrezorTransaction.ts

```diff
@@ -1,33 +1,19 @@
 import TrezorConnect from "@trezor/connect-web";
 import transformTransaction from "@trezor/connect-plugin-stellar";
-import { Transaction } from "stellar-sdk";
-import { loadPrivateKey } from "helpers/keyManager";
+import { Transaction } from "@stellar/stellar-sdk";
 
 export const signTrezorTransaction = async (
   transaction: Transaction,
-  keyStore: any,
+  publicKey: string,
+  bipPath: string,
 ) => {
-  const { keyStoreId, password, custom } = keyStore;
-
-  if (!custom || !custom.email || !custom.appUrl) {
-    throw new Error(
-      `Trezor Connect manifest with "email" and "appUrl" props is required.
-      Make sure they are passed through "custom" prop.`,
-    );
-  }
-
-  const key = await loadPrivateKey({
-    id: keyStoreId,
-    password,
-  });
-
   TrezorConnect.manifest({
-    email: custom.email,
-    appUrl: custom.appUrl,
+    email: "accounts+trezor@stellar.org",
```

Comment:
Is this meant to be hardcoded?

Reply:
Yes, we use the same approach [here](https://github.com/stellar/account-viewer-v2/blob/master/src/components/SignIn/SignInTrezorForm.tsx#L55:L61). Once we confirm that this work, we can create a config file and use the same reference in both places.